### DISCUSSION TITLE
SDXL multi-gpu training

### DISCUSSION THREAD

**Discussion by moonshinegloss** [kohya-ss/sd-scripts#1218] [Q&A]
all mentions of DDP have been wiped from the readme and docs, so I assume it got superseded by deepspeed? whats the config I should do in accelerate for 4090+3090? and what flags do I need to pass for it to work in the training script? thanks.

**Comment by kohya-ss** [comment]
DDP is supported. Sorry for the lack of documentation. Release notes have the documentations.

**Comment by moonshinegloss** [comment]
I have tried to setup DDP via accelerate and launching it this way, but it just gets stuck indefinitely, are there any arguments I need to pass for this to work? thanks
